Examples/Examples-iOS/IGListKitExamples/SectionControllers/MonthSectionController.swift

```diff
@@ -15,7 +15,7 @@
 import UIKit
 import IGListKit
 
-final class MonthSectionController: IGListBindingSectionController, IGListBindingSectionControllerDataSource, IGListBindingSectionControllerSelectionDelegate {
+final class MonthSectionController: IGListBindingSectionController<IGListDiffable>, IGListBindingSectionControllerDataSource, IGListBindingSectionControllerSelectionDelegate {
```

Comment:
Just realized we should have probably used `Month` in here, right? That way the whole class uses `Month` as its type for `object`. I auto landed this so we can follow up.